plugins/modules/source_control/gitlab/gitlab_group_members.py

```diff
@@ -32,15 +32,32 @@
         type: str
     gitlab_user:
         description:
-            - The username of the member to add to/remove from the GitLab group.
-        required: true
-        type: str
+            - A username or a list of usernames to add to/remove from the GitLab group.
+        type: list
```

Comment:
I think this should be string. can we change the description & the type to the original?